src/jit/importer.cpp

```diff
@@ -2878,24 +2878,36 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
                                        CORINFO_SIG_INFO *       sig,
                                        int                      memberRef,
                                        bool                     readonlyCall,
-                                       CorInfoIntrinsics *      pIntrinsicID)
+                                       CorInfoIntrinsics *      pIntrinsicID,
+                                       bool *                   pMustExpand)
 {
-    CorInfoIntrinsics intrinsicID = info.compCompHnd->getIntrinsicID(method);
+    CorInfoIntrinsics rawIntrinsicID = info.compCompHnd->getIntrinsicID(method);
```

Comment:
Check with @pgavlin he has recently done it.  AFAIK, such a change you would have to do it on tfs side, since they are many copies of jit-EE interface definition.